ruby/introduction/how_this_course_will_work.md

```diff
@@ -13,7 +13,7 @@ You'll do a healthy chunk of learning in this course, but by the end of it all,
 Some people believe you can just dive right into Rails and start firing out websites. So why do we have an entire course dedicated to only Ruby before we start looking at Rails? Rails is a framework built using exclusively Ruby, and every bit of code in it is written with Ruby. When (*not* "if") something in your project breaks, you'd better be able to debug it, which means having a strong understanding of Ruby, how it works, and what its error messages look like. Beyond that, when you start to stretch your wings and do something just a bit beyond what the vanilla tutorials explicitly show you how to do, you'll quickly discover a world of frustration if you don't have a strong foundation in Ruby. The amount of time you would spend Googling your error messages and staring blankly at help docs would be better spent learning Ruby now.
 
 ### The Path Ahead
-Ruby is a big language with lots of syntax, so we've broken the concepts up into smaller chunks to make it more digestible. Some of the early material will be fairly straightforward and will rely on simple exercises to help reinforce your understanding. As you progress through the course, the goal is for you to get a much deeper and more practical understanding of the material. As you move into some of the more advanced topics, you'll be reading less and building more--just the way it should be!
+Ruby is a big language with lots of syntax, so we've broken the concepts up into smaller chunks to make it more digestible. Some of the early material will be fairly straightforward and will rely on simple exercises to help reinforce your understanding. As you progress through the course, the goal is for you to get a much deeper and more practical understanding of the material. As you move into some of the more advanced topics, you'll be reading less and building more-- just the way it should be!
```

Comment:
For the change that was made here, do you think it might be better to remove the double hyphens and use one with whitespace on either side (or an em/endash; can't remember which one might be applicable if either would be)? E.g.:

```suggestion
Ruby is a big language with lots of syntax, so we've broken the concepts up into smaller chunks to make it more digestible. Some of the early material will be fairly straightforward and will rely on simple exercises to help reinforce your understanding. As you progress through the course, the goal is for you to get a much deeper and more practical understanding of the material. As you move into some of the more advanced topics, you'll be reading less and building more - just the way it should be!
```

Reply:
> For the change that was made here, do you think it might be better to remove the double hyphens and use one with whitespace on either side (or an em/endash; can't remember which one might be applicable if either would be)? E.g.:

Thanks for your feedback! I think the best option would actually be a single dash with no spaces on either side. Feels a bit odd but a bit of punctuation research tells me this is right.